inventory/byo/hosts.ose.example

```diff
@@ -251,6 +251,9 @@ openshift_master_identity_providers=[{'name': 'htpasswd_auth', 'login': 'true',
 #openshift_cloudprovider_openstack_region=region
 #openshift_cloudprovider_openstack_lb_subnet_id=subnet_id
 #
+# Note: If you're getting a "BS API version autodetection failed" when provisioning cinder volumes you may need this setting
```

Comment:
I think this note could be a little clearer.

Let's include the affected versions as mentioned in kubernetes/kubernetes.github.io#5925

Or perhaps we should wait to see what the approved text is from kubernetes, if that patch ships soon.